ansible/roles/rrdstats/tasks/main.yml

```diff
@@ -0,0 +1,99 @@
+---
+- name: Install rrdtool
+  ansible.builtin.package:
+    name: rrdtool
+    state: present
+  tags:
+    - role::rrdstats
+
+- name: Template the statistics script
+  ansible.builtin.template:
+    src: generate-rrdtool-stats.sh.j2
+    dest: "{{ rrdstats_script_path }}"
+    owner: root
+    group: root
+    mode: "0555"
+  tags:
+    - role::rrdstats
+
+- name: Configure PostgreSQL user
+  become: true
+  become_user: postgres
+  community.postgresql.postgresql_user:
+    comment: Python Discord RRDTool-based longterm statistic gatherer
+    conn_limit: 2
+    name: "{{ rrdstats_pg_username }}"
+    password: "{{ rrdstats_pg_password }}"
+    state: present
+  tags:
+    - role::rrdstats
+
+- name: Configure PostgreSQL privileges to connect to database
+  become: true
+  become_user: postgres
+  community.postgresql.postgresql_privs:
+    database: metricity
```

Comment:
this could also go under postgres_grants in pg role's vars, like we do with the rest.